internal/docs/jakartaee/J2EE14SpecAssertions.xml

```diff
@@ -526,7 +526,7 @@ independent of authentication.
 		</assertion>
 		<assertion required = "true" impl-spec = "false" defined-by = "platform" status = "active" testable = "true">
 			<id>J2EE:SPEC:39</id>
-			<description>Servlets and JSP pages demarcate a transaction using the javax.transaction.UserTransaction interface which is defined in the JTA specification.</description>
+			<description>Servlets and JSP pages demarcate a transaction using the jakarta.transaction.UserTransaction interface which is defined in the JTA specification.</description>
```

Comment:
I wonder why github still shows the javax.transaction => jakarta.transaction changes, as these were already applied.

Reply:
@Tomas-Kraus other than it being April 1 (e.g. a github prank/joke), I'm not sure as to why github shows you changing the same files that I think that I already did change, with regard to javax.transaction => jakarta.transaction.

Reply:
something went wrong I think, when you pulled my change in, as my already merged commit is now shown in the list of commits included by this PR.

Reply:
it's ok, i rebased my changes on top of the master but your changeset has later date than mine. So it's shown here too.

Reply:
just ignore it, git will merge it propery

Reply:
I'm not sure of what git will do, maybe your right but I cannot tell for sure.

I tried cherry-picking your 9 commits on a new branch off master, squashed them into 1 commit, looked at the result and I see unexpected changes like:
`
diff --git a/src/com/sun/ts/tests/common/vehicle/ejb3share/UserTransactionWrapper.java b/src/com/sun/ts/tests/common/vehicle/ejb3share/UserTransactionWrapper.java
index fd147a65..6245983f 100644
--- a/src/com/sun/ts/tests/common/vehicle/ejb3share/UserTransactionWrapper.java
+++ b/src/com/sun/ts/tests/common/vehicle/ejb3share/UserTransactionWrapper.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2007, 2020 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2007, 2018 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
  * terms of the Eclipse Public License v. 2.0, which is available at
@@ -24,10 +24,10 @@ import com.sun.ts.lib.util.TestUtil;

 import javax.persistence.EntityTransaction;
 import javax.persistence.PersistenceException;
-import jakarta.transaction.NotSupportedException;
-import jakarta.transaction.SystemException;
-import jakarta.transaction.UserTransaction;
-import jakarta.transaction.Status;
+import javax.transaction.NotSupportedException;
+import javax.transaction.SystemException;
+import javax.transaction.UserTransaction;
+import javax.transaction.Status;
`
So, either you need to update the pr or someone else that understands how git will fix the pr and not create a problem in the TCK repo, will merge it.  

Perhaps leaving the 6d2c31e7cbe76b08249b908a4d1c4517c46706ca (Reverting jakarta.transaction.xa back to javax.transaction.xa. Not available yet) + 2643796ebb00ccb2ca134fd6f0049c36e4816a1f (Update javax to jakarta for jakarta.transaction) commits out of the new branch would work better.

Reply:
I'll clean it up tomorrow. I can apply all my patches on the top of the master, there is no conflict.
I was reverting jakarta.transaction change in some sources, this is probably ok. But it shall not be seen in the combined diff.

Reply:
Looks like it's not possible to apply those change sets on the top of current master branch.